lib/reducers/create-otp-reducer.js

```diff
@@ -312,6 +327,33 @@ function createOtpReducer (config) {
             }
           }
         })
+      case 'SET_PENDING_REQUESTS':
+        return update(state, {
+          searches: {
+            [searchId]: {
+              pending: { $set: action.payload.pending }
+            }
+          }
+        })
+      case 'SET_ACTIVE_ITINERARIES':
+        const responseUpdate = {}
+        Object.entries(action.payload.assignedItinerariesByResponse)
```

Comment:
I have a slight preference for iterating over `Object.keys` here rather than `Object.entries`, just because we have a habit of iterating over keys much more often.

Reply:
Is there a point in iterating over keys if not using them?

Reply:
I think the use of `Object.entries` is appropriate and clean in this instance.